src/Symfony/Component/JsonPath/Tokenizer/JsonPathTokenizer.php

```diff
@@ -158,6 +187,7 @@ public static function tokenize(JsonPath $query): array
             throw new InvalidJsonPathException('unclosed string literal.', $length - 1);
         }
 
+        $current = trim($current);
         if ('' !== $current) {
```

Comment:
I skipped similar suggestions but this is the preferred CS for such lines:
```suggestion
        if ('' !== $current = trim($current)) {
```